<?xml version="1.0"?>
<!-- ROS head file to simulate vizzy in gazebo -->
<launch>

  <!--<arg name="vizzy_machine_address" default="10.0.29.186"/>-->
  <!--<arg name="env-loader" default="/opt/ros/indigo/env.sh"/>-->
  <!--<machine name="$(arg vizzy_machine_name)" address="$(arg vizzy_machine_address)" env-loader="$(arg env-loader)" user="vizzy"/>-->
  <arg name="left_image_topic" default="/vizzy/l_camera/image_raw"/>
  <arg name="right_image_topic" default="/vizzy/r_camera/image_raw"/>
  <arg name="joint_states" default="/vizzy/joint_states"/>
  <!-- launch stereo_calibration -->
  <node name="stereo_calibration" pkg="online_stereo_calibration_ros" type="perfect_stereo_calibration_node" respawn="true" output="screen" >
    <rosparam file="$(find foveated_stereo_ros)/config/frames.yaml" command="load" />
    <rosparam file="$(find foveated_stereo_ros)/config/params.yaml" command="load" />
    <remap from="left_image" to="$(arg left_image_topic)"/>
    <remap from="right_image" to="$(arg right_image_topic)"/>
    <remap from="joint_states" to="$(arg joint_states)"/>
  </node>

</launch>
